frame/evm/src/lib.rs

```diff
@@ -119,15 +119,15 @@ impl Precompiles for () {
 static ISTANBUL_CONFIG: Config = Config::istanbul();
 
 /// EVM module trait
-pub trait Trait: frame_system::Trait + pallet_timestamp::Trait {
+pub trait Trait: frame_system::Trait + pallet_timestamp::Trait + pallet_balances::Trait {
```

Comment:
```suggestion
pub trait Trait: frame_system::Trait + pallet_timestamp::Trait + pallet_balances::Trait {
```

Why have you included `pallet_balances`?

Reply:
Removed it not sure why!